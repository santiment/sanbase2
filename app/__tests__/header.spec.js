/* eslint-env jest */
import React from './../node_modules/react'
import { mount } from 'enzyme'
import renderer from 'react-test-renderer'
import Header from '../components/header-page'

describe('Header', () => {
  it('Header should render correct name', () => {
    const app = mount(<Header name='check' />)
    expect(app.find('h1').text()).toEqual('check')
  })

  it('Header should render correctly', () => {
    const component = renderer.create(<Header />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
})
