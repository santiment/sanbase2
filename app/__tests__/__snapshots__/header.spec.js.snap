// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header should render correctly 1`] = `
<div
  className="jsx-3745484964 row"
>
  <div
    className="jsx-3745484964 col-lg-12"
  >
    <h1
      className="jsx-3745484964"
    />
    <p
      className="jsx-3745484964"
    />
  </div>
</div>
`;
